src/app/components/autocomplete/autocomplete.component.ts

```diff
@@ -100,10 +105,6 @@ export class AutocompleteComponent implements OnInit, OnDestroy, ControlValueAcc
 		this.onTouched = fn;
 	}
 
-	onSelectionChange(value: string): void {
-		this.onChange(value);
-	}
-
```

Comment:
This has been removed because the method `onOptionSelected` is also triggered when we select a package